paddle/phi/core/dense_tensor_impl.cc

```diff
@@ -354,6 +359,22 @@ std::vector<DenseTensor> DenseTensor::Chunk(int64_t chunks,
   return Split(split_size, axis);
 }
 
+#ifdef PADDLE_WITH_MKLDNN
+dnnl::memory::desc DenseTensor::mem_desc() const {
+  return mem_desc_
+             ? mem_desc_
+             : dnnl::memory::desc(
+                   phi::vectorize(meta_.dims),
+                   paddle::framework::ToMKLDNNDataType(
```

Comment:
please add a new ToMKLDNNDataType method in phi/core/compat/convert_utils.h, it trans `phi::DataType` into `MKLDNNDataType`, the `phi/core` impls cannot depends the fluid framework proto, so here cannot call `paddle::framework::TransToProtoVarType` and include `"paddle/fluid/framework/convert_utils.h"`

Reply:
> I have added such a method to avoid including `"paddle/fluid/framework/convert_utils.h"`

Hi, @chenwhql , @jakpiase Has added the method as you suggested. Could you please review approve this PR? And all other CIs passed. Thanks!